flask-backend/api/routes/user.py

```diff
@@ -105,15 +105,6 @@ def roleupdate():
 @user.route('/delete', methods=['POST'])
 @login_required
 def deleteuser():
-
-    # Check if email is provided or not
-    try:
-        req = request.get_json()
-        email = str(req['email'])
-    except:
-        return 'please provide email', 400
-
-    user = User.query.filter_by(email=email).first()
-    db.session.delete(user)
+    db.session.delete(current_user)
```

Comment:
this API will be used by the admin to delete any particular user, we're not supposed to delete the current user here.

Reply:
@shivanshu1333 Yeah that's what I was also thinking. You approved #68  so I thought we're changing the functionality. I'll change it back.

Reply:
Hi @shivanshu1333, @amlannandy, 
Previously I also thought that only admin can delete any of his user, but at that time some funtionality related to admin was not created like create user by admin, route to show all users of admin, so thought to change delete function in such a way that only a user can delete only his account as previously any user can delete anybody's account.
I have also mentioned a issue related to this, I'll soon raise a PR for that, it would contain route for only admin can create users and delete his users.

Reply:
@sachinsom93 I see. Sounds good. 👍🏻